net/sshuttle/Makefile

```diff
@@ -0,0 +1,72 @@
+#
+# Copyright (C) 2022 Thomas Cujé
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=sshuttle
+PKG_VERSION:=1.1.1
+PKG_RELEASE:=1
+
+PYPI_NAME:=$(PKG_NAME)
+PKG_HASH:=f5a3ed1e5ab1213c7a6df860af41f1a903ab2cafbfef71f371acdcff21e69ee6
+
+PKG_MAINTAINER:=Thomas Cujé <dev.cuje@freenet.de>
+PKG_LICENSE:=LGPL-2.1-only
+PKG_LICENSE_FILES:=LICENSE
+
+include ../../lang/python/pypi.mk
+include $(INCLUDE_DIR)/package.mk
+include ../../lang/python/python3-package.mk
+
+define Package/sshuttle
+  SUBMENU:=SSH
+  SECTION:=net
+  CATEGORY:=Network
+  TITLE:=sshuttle: where transparent proxy meets VPN meets ssh
+  URL:=https://github.com/sshuttle/sshuttle
+  EXTRA_DEPENDS:= \
+  	python3-light \
+  	python3-ctypes \
+  	python3-urllib
+  PKGARCH:=all
+endef
+
+define Package/sshuttle/description
+Transparent proxy server that works as a poor man's VPN. Forwards over ssh. Doesn't require admin. Works with Linux and MacOS. Supports DNS tunneling.
+endef
+
+define Package/sshuttle/conffiles
+/etc/config/sshuttle
+endef
+
+# Some sshuttle script files are copied to SSH remote hosts and are therefore mandatory.
+# This causes the otherwise pre-compiled package to contain the source package.
+# Compare with https://github.com/sshuttle/sshuttle/blob/v1.1.1/sshuttle/ssh.py#L95-L103
+# Since the source is very small in general, to reduce complexity
+# and support all architectures no explicit filtering is used.
+define Package/sshuttle/install
```

Comment:
Which script files are needed exactly? By shipping the Python source files instead of precompiled bytecode, this forces target Python to compile the source files on device. This can be very slow and the reason why we ship bytecodes in the default packages.

Reply:
I debugged into it and it's just the file listed there in the ssh.py source code for this version of sshuttle, but it may change in the future.
The files are needed in the first place to be sent and compiled on the SSH target device, not the OpenWRT device. To avoid compilation on the OpenWRT device, we would need to ship BOTH compiled and script version of the files, which I tried, but duplicates the required disk space, which is way worse IMO.